var/spack/repos/builtin/packages/r-limma/package.py

```diff
@@ -33,7 +33,8 @@ class RLimma(RPackage):
     url      = "https://git.bioconductor.org/packages/limma"
     list_url = homepage
 
+    version('3.34.9', git='https://git.bioconductor.org/packages/limma', commit='6755278a929f942a49e2441fb002a3ed393e1139')
     version('3.32.10', git='https://git.bioconductor.org/packages/limma', commit='593edf28e21fe054d64137ae271b8a52ab05bc60')
     version('3.32.6', 'df5dc2b85189a24e939efa3a8e6abc41')
 
-    depends_on('r@3.4.0:3.4.9', when='@3.32.10')
+    depends_on('r@3.4.0:3.4.9', when='@3.34.9')
```

Comment:
you don't want to remove the dependency from the old version. If this is a dependency for all future versions do something like `when='@3.32.10:'`

Reply:
will do that